src/tokens/ods/space.yaml

```diff
@@ -3,7 +3,7 @@ space:
   xsmall:
     value: 4px
   small:
-    value: 8px
+    value: 7px
```

Comment:
Uff.. is this needed? I remember that there are some size calculations in web based on the assumption that we have an 8px grid.